modules/base/mw.cx.SiteMapper.js

```
@@ -208,8 +207,8 @@ mw.cx.SiteMapper.prototype.getLanguagePairs = function () {
 			.then((response) => response.json())
 			.then((response) => ({
 				targetLanguages: response.target,
-				sourceLanguages: ["en", "ar"],
-				// sourceLanguages: response.source,
+				sourceLanguages: response.source,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)